general/administration/connectivity.md

```diff
@@ -18,6 +18,26 @@ If you can access the server locally but not outside of your LAN, then you likel
 
 The easiest way to check for issues is by checking the logs, which can be accessed through the settings on the web frontend or in the log directory on your server. If there are no logs at all relating to web traffic, even over a LAN connection, then the server hasn't been reached at all yet. This would indicate either an incorrect address or an issue somewhere else on the network.
 
+## Debug Mode
```

Comment:
```suggestion
## Debug (more verbose) Logging
```
?

Reply:
if someone doesn't know what debug means, they might not know what verbose means lol

Reply:
I'm still getting used to doing PR's on git. Let me know if I need to do anything to get this or my others merged.

Reply:
This is not a debug mode per se, this is extensive logging. Debugging mode usually means your application is better suited for interoperating with a debugger tool like gdb, which is not the case here.

Reply:
makes sense

Reply:
> I'm still getting used to doing PR's on git. Let me know if I need to do anything to get this or my others merged.

You have to make changes and push them to your branch for them to show up here, then we can review and merge them.